app/src/main/java/io/appium/settings/receivers/LocaleSettingReceiver.java

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright 2012-present Appium Committers
+  <p>
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  <p>
+  http://www.apache.org/licenses/LICENSE-2.0
+  <p>
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+package io.appium.settings.receivers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import java.util.Locale;
+
+import io.appium.settings.handlers.LocaleSettingHandler;
+
+public class LocaleSettingReceiver extends BroadcastReceiver {
+    private static final String TAG = LocaleSettingReceiver.class.getSimpleName();
+
+    private static final String LANG = "LANG";
+    private static final String COUNTRY = "COUNTRY";
+
+    // am broadcast -a io.appium.settings.locale --es LANG ja --es COUNTRY JP
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String language = intent.getStringExtra(LANG);
```

Comment:
Please add error handling. We should not crash the whole service if the input is incorrect